README.md

```diff
@@ -17,44 +18,98 @@ In contrast, our philosophy is rooted in simplicity:
 
 We believe this approach is the best way to build a client that is both fast and resilient. By adhering to these principles, we will be able to iterate fast and explore next-generation features early, either from the Ethereum roadmap or from innovations from the L2s.
 
-## Usage
+## Quick Start (localnet)
+
+![Demo](https://raw.githubusercontent.com/lambdaclass/ethereum_rust/8e3b69d727225686eec30b2c2b79cecdf7eac2d9/Demo.png)
+
+### Prerequisites
+- [Kurtosis](https://docs.kurtosis.com/install/#ii-install-the-cli)
+- Rust.
+- Docker.
+```shell
+make localnet
+```
+
+This make target will:
+1. Build our node inside a docker image.
+2. Fetch our fork [ethereum package](https://github.com/ethpandaops/ethereum-package), a private testnet on which multiple ethereum clients can interact.
+3. Start the localnet with kurtosis.
+
+If everything went well, you should be faced with our client's logs (ctrl-c to leave)
+
+To stop everything, simply run:
+```shell
+make stop-localnet
+```
 
+
+## Dev Setup
 ### Build
 
-To build the main executable and its crates, run:
+#### Rust
+To build the node, you will need the rust toolchain: 
+1. First, [install asdf](https://asdf-vm.com/guide/getting-started.html): 
+2. Add the rust plugin: 
+```shell
+asdf plugin-add rust https://github.com/asdf-community/asdf-rust.git
+```
+3. cd into the project and run:
+```shell
+asdf install
+```
+
+You now should be able to build the client:
 ```bash
 make build
 ```
 
-### Test
-Note: To execute EF tests, the test fixtures are required. To download them, run:
+### Testing
+
+For testing, we're using three kinds of tests.
+
+##### Ethereum Foundation Tests
+
+These are the official execution spec tests, tests that should be implementation agnostic, you can execute them with:
+
 ```bash
-make download-test-vectors
+make test
 ```
 
-To run the tests from a crate, run:
+This will previously download it from the [official execution spec tests repo](https://github.com/ethereum/execution-spec-tests/) and run them with our glue code
```

Comment:
```suggestion
This will download the test cases from the [official execution spec tests repo](https://github.com/ethereum/execution-spec-tests/) and run them with our glue code
```
nit